mods/content/matchmaking/matchmaker.dm

```diff
@@ -271,8 +271,8 @@ var/global/datum/matchmaker/matchmaker = new()
 			var/list/relations = matchmaker.get_relationships(mind)
 			for(var/datum/relation/R in relations)
 				R.finalize()
-			show_browser(usr,null, "window=relations")
+			show_browser(src,null, "window=relations")
```

Comment:
oh good catch, yeah since it's OnSelfTopic this is always safe, but that's not necessarily true in other cases. i'll make sure to keep that in mind when rewriting Topic stuff to use OnTopic later